ports/comms/portfile.cmake

```
@@ -1,56 +1,23 @@
 #header-only library
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    REPO commschamp/comms_champion
-    REF v3.4
-    SHA512 573afbc0aebd72d8a047067410f0f54588675c4cbad37f824edbb6d8303e9c191c573ac9325bf5fec575dffd3d05562c04e75c1e5b748a34d01056bc8b766fb1
+    REPO commschamp/comms
+    REF v4.0
+    SHA512 ec83bef647dd6c32e6ba98ce51970c48befaa2b0ff9c26f538fb0ce72e46da14cd592a0c652af5f9f10906f7058ff623dcf13ac4b81c96c0aea1fd8a31551bb7
     HEAD_REF master
 )
 
-vcpkg_check_features(
-    OUT_FEATURE_OPTIONS FEATURE_OPTIONS
-    FEATURES
-        tools   CC_BUILD_TOOLS_LIBRARY
-        tools   CC_INSTALL_TOOLS_LIBRARY
-        tools   CC_BUILD_TOOLS
-        tools   CC_INSTALL_TOOLS
-)
-
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
-        ${FEATURE_OPTIONS}
-        -DCC_INSTALL_COMMS_LIB=ON
-        -DCC_BUILD_UNIT_TESTS=OFF
-        -DCC_WARN_AS_ERR=OFF
-        -DCC_BUILD_DEMO_PROTOCOL=OFF
-        -DCC_INSTALL_DEMO_PROTOCOL=OFF
+        -DCC_COMMS_BUILD_UNIT_TESTS=OFF
+        -DBUILD_TESTING=OFF

```

```suggestion
        -DBUILD_TESTING=OFF
        -DCC_WARN_AS_ERR=OFF
```
vcpkg policy is that warnings should never be made errors for package builds (it breaks users for no good reason -- warnings are for the actual authors of the library)